pkgs/applications/misc/oil/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenvNoCC, lib, fetchFromGitHub, jq, gawk, peco, makeWrapper }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "oil-buku";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "AndreiUlmeyda";
+    repo = "oil";
+    rev = version;
+    sha256 = "12g0fd7h11hh94b2pyg3pqwbf8bc7gcnrnm1qqbf18s6z02b6ixr";
+  };
+
+  postPatch = ''
+    substituteInPlace src/oil --replace \
+      "LIBDIR=/usr/local/lib/oil" "LIBDIR=${placeholder "out"}/lib"
+
+    substituteInPlace Makefile --replace \
+      "LIBDIR ?= /usr/local/lib/oil" "LIBDIR ?= ${placeholder "out"}/lib" \
+
+    substituteInPlace Makefile --replace \
+      "BINDIR ?= /usr/local/bin" "BINDIR ?= ${placeholder "out"}/bin"
```

You could do this with one `substituteInPlace`:
```suggestion
    substituteInPlace Makefile \
      --replace "LIBDIR ?= /usr/local/lib/oil" "LIBDIR ?= ${placeholder "out"}/lib" \
      --replace "BINDIR ?= /usr/local/bin" "BINDIR ?= ${placeholder "out"}/bin"
```